core/arch/arm/kernel/thread_optee_smc_a32.S

```diff
@@ -161,7 +161,9 @@ DECLARE_KEEP_PAGER thread_vector_table
 
 FUNC thread_std_smc_entry , :
 UNWIND(	.cantunwind)
+	push	{r4, r5}
```

Comment:
Maybe add `/* arm32 calling convention args 5 and 6 passed on the stack */`?

Reply:
Yes sorry, I had it in the changed buffer in my editor.